js/ui/appDisplay.js

```diff
@@ -493,18 +497,16 @@ class AppDisplay extends BaseAppView {
             if (!this._currentDialog.contains(actor))
                 this._currentDialog.popdown();
         });
-        Main.overview.addAction(this._clickAction, false);
+        if (params.addBackgroundAction)
+            Main.overview.addAction(this._clickAction, false);
         this._eventBlocker.bind_property('reactive', this._clickAction,
             'enabled', GObject.BindingFlags.SYNC_CREATE);
 
         this._bgAction = new Clutter.ClickAction();
-        Main.overview.addAction(this._bgAction);
-        BackgroundMenu.addBackgroundMenuForAction(this._bgAction, Main.layoutManager);
-        this._clickAction.bind_property('enabled', this._bgAction,
```

Comment:
Should the bg action still be enabled if the click action is enabled? (e.g. when the folder popup is opened?).